source/adapters/cuda/device.cpp

```diff
@@ -1008,8 +1008,10 @@ UR_APIEXPORT ur_result_t UR_APICALL urDeviceGetInfo(ur_device_handle_t hDevice,
           &MemoryBusWidth, CU_DEVICE_ATTRIBUTE_GLOBAL_MEMORY_BUS_WIDTH,
           hDevice->get()));
     }
+    uint64_t MemoryBandwidthConstant = 250;
 
-    uint32_t MemoryBandwidth = MemoryClockKHz * MemoryBusWidth * 250;
+    uint64_t MemoryBandwidth =
```

Comment:
If this is bits per second, don't we need to multiply by 1000 to account for the kHz aspect?

Reply:
Also where does the `MemoryBandwidthConstant` come from? I know it was already there, but have providence comments for magic numbers would be useful.

Also, since it's putatively a constant: `const` or `constexpr`, please

Reply:
I think the original equation is meant to use it as KHz to get it as b/s. I actually changed the constant as I think 250 was reporting it as Byte/sec and it should be b/s as the sycl spec. I added a comment explaining how to get this constant from in the nvidia doc page. Do you think that should be enough or should I write the computation constants also in a comment?